fapi-client/src/main/scala/com/gu/facia/api/models/collection.scala

```diff
@@ -83,7 +98,21 @@ object Collection {
       collection.live
       .filter(_.isSnap)
       .filter(_.safeMeta.snapType == Some("latest"))
-      .flatMap(snap => snap.safeMeta.snapUri.map(uri => snap.id -> uri)).toMap)
+      .flatMap(snap => snap.safeMeta.snapUri.map(uri => snap.id -> uri))
+      .toMap)
+
+  def treatsRequestFor(collection: Collection): TreatsRequest = {
```

Comment:
I don't much like this function. It's only used in one place, so why not inline it somewhere? Even returning a tuple might make it a bit simpler. This is wrapped up with my comment above about the logic in the FAPI method though, so maybe it needs a little thought up there and that'll make all this stuff simpler as well.